packages/frontend-core/src/api/index.js

```diff
@@ -170,6 +170,14 @@ export const createAPIClient = config => {
 
     // Handle response
     if (response.status >= 200 && response.status < 400) {
+      const migration = response.headers.get("migration-header")
```

Comment:
We should get @aptkingston thoughts on best way to handle this, I think an event dispatcher which the builder can attach to could work, but he might have a better idea!